tests/unit/components/Scorecard.spec.ts

```diff
@@ -96,6 +97,13 @@ describe("Scorecard.vue", () => {
       isShown: true,
     });
     await flushPromises();
+    const confetti = require("canvas-confetti");
+    const confettiCanvas = document.getElementById("confetticanvas");
+    const confettiHandler = confetti.create(confettiCanvas, {
+      resize: true,
+    });
+    throwConfetti(confettiHandler);
```

Comment:
we have discussed so many times that this is not how you trigger methods in test no? Then, why are you repeating this same mistake?

Reply:
We write await before the method in testing.
![image](https://user-images.githubusercontent.com/73693628/164461123-73f02017-e856-452f-9a74-a6366d10010d.png)